packages/patternfly-3/patternfly-react/src/components/DateTimePicker/TimePicker.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormControl, InputGroup, Icon, OverlayTrigger, Popover } from '../../';
+import TimeInput from './TimeComponents/TimeInput';
+import { getDateFromTime, formatTime } from './TimeHelpers';
+
+class TimePicker extends React.Component {
+  state = {
+    value: getDateFromTime(this.props.value),
+    tmpValue: formatTime(this.props.value, this.props.locale),
+    hiddenValue: this.props.hiddenValue
+  };
+
+  setSelected = time => {
+    const newTime = getDateFromTime(time);
+    this.setState({ value: newTime, tmpValue: formatTime(newTime, this.props.locale) });
+  };
+  render() {
+    const { value, hiddenValue } = this.state;
+    const { locale, placement, id } = this.props;
+    const popover = (
+      <Popover
+        id={id}
+        className="bootstrap-datetimepicker-widget date-time-picker-pf bootstrap-timepicker-widget  dropdown-menu"
```

Comment:
```suggestion
        className="bootstrap-datetimepicker-widget date-time-picker-pf bootstrap-timepicker-widget"
```
Adding the `dropdown-menu` class is causing the popover to be incorrectly placed at absolute left 0. I don't believe we need that class for any other reason.

Reply:
removed. now the popover is above/under the clock icon and the arrow is pointing at the clock icon